### DISCUSSION TITLE
Input/output error - Error writing file

### DISCUSSION THREAD

**Discussion by cyanoluu** [bbuchfink/diamond#623] [Q&A]
Hello,

I would like to ask you if you have any experience with this error:

Computing hash join...  [36.871s]
Masking low complexity seeds...  [6.772s]
Searching alignments... Input/output error
terminate called after throwing an instance of 'File_write_exception'
  what():  Error writing file diamond-tmp-xhJ9lW
/var/spool/pbs/mom_priv/jobs/13257712.meta-pbs.metacentrum.cz.SC: line 37: 414132 Aborted                 diamond blastx -d nr -b 12 -c 1 -q $DATADIR/met115/reads-met115.fastq -o met115-diamond.daa -f 100
Calculation ended up erroneously (with a code 134) !!

Thanks a lot,
L

**Comment by bbuchfink** [comment]
It points to some problem of writing to your output directory, a more specific reason is not given.
